build/release-in-a-container.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Check and error if not "in-a-container"
+if [[ ! -f /.dockerenv ]]; then
+  echo
+  echo "'make release-in-a-container' can only be used from a GCB container!"
+  echo
+  exit 1
+fi
+
+# First run "make cross" forcing output to 'dockerized' (see top level Makefile)
```

Comment:
I think you can run 'make cross' as part of the release, right?
no need to run 2 steps

Reply:
The idea is to provide a similar entry point from the top level Makefile to keep it to a single step.  So users or the release tooling can just call 'make release' or 'make release-in-a-container'.

Reply:
if you see the original 'make release' script, as part of the script it will call the build_image and then the make cross.
https://github.com/kubernetes/kubernetes/blob/master/build/release.sh

Reply:
Done. PTAL.